### PR TITLE
simulation tutorial

### PR DESCRIPTION
**Pull request by tomerfqm**
To match the simulator page in the docs

### CODE REVIEW

File: Tutorials/intro-to-simulation/intro-to-simulation.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+"""
+import numpy as np
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm import SimulationConfig
+from qm import LoopbackInterface
+from qm.qua import *
+import matplotlib.pyplot as plt
+from configuration import *
+
+qmm = QuantumMachinesManager()
```
Comment (yomach) [inline Tutorials/intro-to-simulation/intro-to-simulation.py:15]: Maybe add IP?

File: Tutorials/intro-to-simulation/intro-to-simulation.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+# ## Example 1 - Basic simulation, No inputs
+
+with program() as prog1:
+    a = declare(fixed)
+    c = declare(fixed, value=0.4)
+
+    with for_(a, 0.2, a < 0.9, a + 0.1):
+        play("const" * amp(a), "qe1")
+
+    save(c, "c")
```
Comment (yomach) [inline Tutorials/intro-to-simulation/intro-to-simulation.py:27]: I think we should use the stream processing and (maybe with `with_timestamps()` and `save_all()`).
Do we encourage users to use this save method?

File: Tutorials/intro-to-simulation/readme.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+how the simulator can simulate saving variables, as would occur in the real hardware. 
+
+Third Example
+==============
+
+The third example demonstrates a slightly more advanced usage. It shows how a loop-back connection can be defined, 
+to simulate acquisition and demodulation of ADC signals. In the example, a connection from analog output 1 of controller 1
+is connected to analog input 1 of controller 1. The example also shows that the demodulation and adc input can be simulated 
+and saved to the stream processing, which later can be fetched and analyzed. It is important to note that the data will only
+be available if the simulation duration was long enough to simulate it. FIn the current example, to "fill" the buffer in the stream processing,
```
Comment (yomach) [inline Tutorials/intro-to-simulation/readme.md:35]: **FIn**

**Review by yomach — COMMENTED** [review]
Only small comments

File: Tutorials/intro-to-simulation/intro-to-simulation.py (hunk 1 above)
Comment (tomerfqm) [reply Tutorials/intro-to-simulation/intro-to-simulation.py:15]: Which IP?

Comment (yomach) [reply Tutorials/intro-to-simulation/intro-to-simulation.py]: A variable `QOP_IP` which will be set to `None` or `127.0.0.1` for now?

**Comment by tomerfqm** [comment]
@yomach fixed and added another example

**Comment by yomach** [comment]
> @yomach fixed and added another example

Amazing.
I wrote a comment saying maybe let's add a multi OPX example, but then deleted it because I didn't wanna give you more tasks :)

**Pull request merged**